.github/workflows/build_test.yml

```diff
@@ -192,6 +192,46 @@ jobs:
           retention-days: 1
           overwrite: true
 
+  mothership-e2etest:
```

Comment:
A question: so this is a dedicated job, and it will run unconditionally (just the label required), am i right? i wonder whether we could run this test conditionally depending on the config, as it is right now for the other providers. for instance, what if want to run only vsphere provider tests on our internal runner? if i, once again, understand correctly, there will be 2 runners engaged: the internal one and GH public runners (please correct me if i wrong), but the latter is highly likely is not the desirable state: initially the only intense was to check the vsphere